src/account_manager.rs

```diff
@@ -426,6 +439,44 @@ impl AccountManager {
         })
     }
 
+    /// Convert the first address from the first account to a migration tryte address
+    pub async fn get_migration_address(&self) -> crate::Result<String> {
```

Comment:
Should this error out if there's no account created?

Reply:
Yes